release notes/v0.43.0.md

```diff
@@ -0,0 +1,124 @@
+
+All current xk6-browser scripts that work with v0.8.0 will continue to work with the built-in module in k6 v0.43.0, by simply changing the import path from `k6/x/browser` to `k6/experimental/browser`, and setting the environment variable `K6_BROWSER_ENABLED` to `true`. You can see a script example [here](https://github.com/grafana/k6/blob/v0.43.0/samples/experimental/browser.js), and read the updated [xk6-browser documentation](https://k6.io/docs/javascript-api/xk6-browser/).
+
+The module is currently under the `experimental` namespace as we reserve the decision to introduce breaking changes in the future, but our mid-term goal is to drop the `experimental` label and make browser support a stable part of the k6 feature set.
+
+
+### Native support for JavaScript's `async`/`await` [#2830](https://github.com/grafana/k6/pull/2830)
+
+In [v0.35.0](https://github.com/grafana/k6/releases/tag/v0.35.0) we added support for asynchronous functionality in k6 scripts with the addition of `Promise`.
+
+This was very useful, but the experience wasn't very friendly, as users had to use the `.then()` API to resolve Promises, instead of the `async`/`await` syntax available in most other JavaScript runtimes since ECMAScript 2016. [Some workarounds were possible](https://github.com/grafana/k6/issues/779#issuecomment-964027280), but it required a separate build pipeline to transpile the syntax into the older ES5.1+ standard supported by k6.
+
+That is, until now! :tada: With the invaluable help from @dop251, the maintainer of [goja](https://github.com/dop251/goja), the JS VM k6 uses, in k6 v0.43.0 it's possible to use `async`/`await` natively in your k6 scripts! This functionality works just as you'd expect in other JS runtimes, and makes working with async APIs much more convenient. See the [`http.asyncRequest()` example](#httpasyncrequest-2877) below for usage details.
+
+One caveat to note is that async functions can't be passed to `group()` or `check()`. This usage is not not compatible with asynchronous behavior, so you will get an error if trying to do so.
+
+
+### Experimental JavaScript module for distributed tracing [#2853](https://github.com/grafana/k6/pull/2853) [#2854](https://github.com/grafana/k6/pull/2854) [#2855](https://github.com/grafana/k6/pull/2855)
+
+This release brings a new experimental JavaScript module that adds [distributed tracing](https://en.wikipedia.org/wiki/Tracing_(software)) support to k6. It allows you to seamlessly instrument HTTP requests from k6, and, assuming the system you're testing is instrumented in the same way, would allow you unparalleled visibility of the SUT behavior for the lifetime of each request.
```

Comment:
Should we expand here with information about how this would be useful? Mention Tempo, etc.?

Reply:
The documentation mentions Tempo as an example of a context in which the module's client could be used. However, we haven't emphasized on the tracing backends and more on the supported formats. Thus, I'd rather recommend keeping these notes agnostic of Tempo and our other tracing-related projects. However, I have literally zero strong opinions on this and would be fine both ways 👍🏻